TestResult.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, description, name=None):
         self.timer = time.time()
 
     def _current_time(self):
-        return str(datetime.datetime.now().time())
+        return str(datetime.datetime.now().time())[:-3]
```

Comment:
I suggest not doing that.

str(datetime.datetime.now().replace(microsecond=0).time()) == '09:35:14'

Reply:
I'm not sure second precision is really enough for this

Reply:
Exactly, that's my point

Reply:
Ok, I thought the docs said it would be zero-padded, but if that's not always the case I'll do it another way.

Reply:
[Return a string representing the time in ISO 8601 format, HH:MM:SS.mmmmmm or, if self.microsecond is 0, HH:MM:SS](https://docs.python.org/2/library/datetime.html#datetime.time.isoformat)

Reply:
What does the following return on Windows? I'd hope that being explicit with the %f would return six zeros if it has nothing else to put there:

```
datetime.datetime.now().strftime("%H:%M:%S.%f")[:-3]
```

Reply:
Works for me everywhere I've tried it. :-)

Reply:
Ok, I'll do that for now, but happy to re-visit if we come across a platform which handles things differently.